postgres/lib.w

```diff
@@ -57,7 +119,167 @@ pub class Database {
     return this.inner.query(query);
   }
   pub inflight connectionOptions(): ConnectionOptions  {
-    return this.inner.connectionOptions();
+    return this.inner.connectionOptions()!;
+  }
+}
+
+struct TfawsApp {
+  vpc: tfaws.vpc.Vpc;
+  subnets: TfawsAppSubnets;
+}
+
+struct TfawsAppSubnets {
+  private: Array<tfaws.subnet.Subnet>;
+  public: Array<tfaws.subnet.Subnet>;
+}
+
+class DatabaseRDS impl IDatabase {
+  var secretRef: aws.SecretRef;
+  var endpoint: str;
+  params: RequiredRDSParameters;
+  databaseName: str;
+  pub connection: ConnectionOptions;
+
+  new (props: DatabaseProps) {
+    this.databaseName = props.name;
+
+    let app = nodeof(this).app;
+    this.params = RequiredRDSParameters.fromJson(app.parameters.read(schema: RequiredRDSParameters.schema()));
+    let tfawsApp: TfawsApp = unsafeCast(nodeof(this).app);
+    let vpc = tfawsApp.vpc;
```

Comment:
Pragmatic 👍  we should see if we can expose the default VPC somehow in the main SDK